keyboards/okuura/goaisatsu/info.json

```
@@ -0,0 +1,127 @@
+{
+    "manufacturer": "okuura",
+    "keyboard_name": "goaisatsu",
+    "maintainer": "okuura",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["F4", "F5", "F6", "F7", "B1", "B3", "B2", "B6"],
+        "rows": ["D4", "C6", "D7", "E6", "B4", "B5"]
+    },
+    "processor": "atmega32u4",
```

pro micro is used (mentioned in `readme.md`)
```suggestion
    "development_board": "promicro",
    "diode_direction": "COL2ROW",
    "features": {
        "bootmagic": true,
        "command": false,
        "console": false,
        "extrakey": true,
        "mousekey": true,
        "nkro": true
    },
    "matrix_pins": {
        "cols": ["F4", "F5", "F6", "F7", "B1", "B3", "B2", "B6"],
        "rows": ["D4", "C6", "D7", "E6", "B4", "B5"]
    },
```